### PR TITLE
89 create desktop display

### PR DESCRIPTION
**Pull request by Dan-Quin**
Meant to send this earlier, but this should be the main rework of the main menu that we went through this morning.

### CODE REVIEW

File: frontend/src/components/DesktopApps/Taskbar.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { IGameData } from "../../../../shared/types";
+import { useDispatch, useSelector } from "react-redux";
+import { IRefPhaserGame, PhaserGame } from "../../game/PhaserGame";
+import UpgradesContainer from "../UpgradesContainer";
+import { sellData } from "../../slices/gameDataSlice";
+import AutoCollector from "../AutoCollector";
+
+interface TaskbarProps {
+  // Add any props you need here
```
Comment (josephmaxwellhyland) [inline frontend/src/components/DesktopApps/Taskbar.tsx:10]: A lot of this stuff can be cleaned up - remove any `import` statements that are not used, etc

**Review by josephmaxwellhyland — CHANGES_REQUESTED** [review]
Overall this looks quite good - just a couple little code cleanup things!

**Comment by jvgusch** [comment]
By the way, I HIGHLY recommend adding screenshots to your PRs when making UI changes. That way someone doesn't have to pull down your branch to see the visual changes you are making. But when someone does it's also very helpful because you can verify that the changes are the same across devices.

**Review by josephmaxwellhyland — APPROVED** [review]
looks good

**Review by josephmaxwellhyland — APPROVED** [review]
good

**Pull request merged**